mgtool/configuration.go

```diff
@@ -3,13 +3,17 @@ package mgtool
 import (
 	"fmt"
 	"strings"
+
+	"github.com/magefile/mage/mg"
 )
 
 const (
 	AMD64 = "amd64"
 	X8664 = "x86_64"
 )
 
+type Prepare mg.Namespace
```

Comment:
is this used to ignore outputting to Makefiles? maybe add as comment?

Reply:
Yeah